src/components/molecules/InputElement/InputElement.tsx

```diff
@@ -1,30 +1,33 @@
+import clsx from "clsx";
 import { RequiredBadge } from "./RequiredBadge";
 import type { InputElementProps } from "./type";
 
 export const InputElement = ({
   children,
-  title,
+  label,
   errorText,
   descriptionText,
+  showLabel = true,
   showRequired,
   isRequired,
   optionalText,
   requiredText,
 }: InputElementProps) => {
+  const visibleLabel = showLabel ? "text-sm font-medium leading-tight" : "sr-only";
+  const visibleGap = !!showLabel && !!showRequired && "gap-x-2";
+
   return (
     <div className="flex flex-col gap-y-2">
-      {(!!title || !!showRequired) && (
-        <div className="flex items-center gap-x-2">
-          {!!title && <span className="text-sm font-medium leading-tight">{title}</span>}
-          {!!showRequired && (
-            <RequiredBadge
-              isRequired={isRequired}
-              optionalText={optionalText}
-              requiredText={requiredText}
-            />
-          )}
-        </div>
-      )}
+      <div className={clsx("flex items-center", visibleGap)}>
+        <label className={visibleLabel}>{label}</label>
```

Comment:
Ah sorry, in my example I completely forgot that we need an `htmlFor` and an `id` to connect the label with the input:
```suggestion
        <label htmlFor={id} className={visibleLabel}>{label}</label>
```
And then add `id` as a prop.
Sorry about that!